server/env.ts

```diff
@@ -20,6 +20,7 @@ const env = cleanEnv(process.env, {
   REDIS_PORT: num({ default: 6379 }),
   REDIS_PASSWORD: str({ default: "" }),
   REDIS_DB: num({ default: 0 }),
+  REDIS_SSL: bool({ default: false }),
```

Comment:
Why use a "deprecated" term like SSL instead of TLS directly?
PS: Some projects use REDIS_PROTOCOL (here you are proposing REDIS_SSL) as a flag with 2 options: (`redis`, `rediss`) as in (`redis://:<password>@<host>:<port>/<db>`, `rediss://:<password>@<host>:<port>/<db>`).